frontend/tests/utils/wmgUtils.ts

```diff
@@ -5,7 +5,23 @@ import { getTestID, getText } from "tests/utils/selectors";
 import { selectFirstOption, tryUntil } from "./helpers";
 import { ADD_GENE_BTN, ADD_TISSUE_BTN } from "../common/constants";
 
-const { skip } = test;
+const { skip, beforeEach } = test;
+
+const WMG_SEED_TISSUES = ["blood", "lung"];
+const WMG_SEED_GENES = ["DPM1", "TNMD", "TSPAN6"];
+
+/**
+ * (thuang): Seed app state with some tissues and genes
+ */
+export const WMG_WITH_SEEDED_TISSUES_AND_GENES = {
```

Comment:
Use share URL to set up WMG state before testing instead of clicking around, wasting time